packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -118,13 +154,29 @@ You can use your custom implementation of the Next Query's content. In the examp
 using the default Next Query's content, an icon is added, as well as a span with the query of the
 Next Query suggestion.
 
-```vue
-<NextQueries>
-  <template #suggestion-content="{suggestion}">
-    <img src="./next-query-icon.svg" class="x-next-query__icon"/>
-    <span class="x-next-query__query">{{ suggestion.query }}</span>
-  </template>
-</NextQueries>
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work